app/src/main/java/de/westnordost/streetcomplete/quests/surface/AddCyclewayPartSurface.kt

```diff
@@ -0,0 +1,29 @@
+package de.westnordost.streetcomplete.quests.surface
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.meta.OsmTaggings
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao
+
+class AddCyclewayPartSurface(o: OverpassMapDataDao) : SimpleOverpassQuestType<String>(o) {
+
+    override val tagFilters = """
+        ways with (
+        (bicycle ~ designated|yes and segregated=yes and highway ~ path|footway|cycleway|bridleway and surface = paved)
+        or
+        (cycleway=track and highway ~ ${OsmTaggings.ALL_ROADS.joinToString("|")})
```

Comment:
This won't work. For cycleways tagged on roads, it would need to be a selection to separately tag the left and the right side.

Reply:
It seems that with such handling it would be possible to ask about `cycleway:left=track`, `cycleway:right=track`, `cycleway:both=track`, but not about `cycleway=track`.

I assumed that `cycleway=track` may have cycleway on one or both sides, but in case of cycleway on both sides it almost always will be the same surface.

Reply:
But sometimes it is not.

Also, especially for broad roads, it is difficult to ask for both sides at the same time implicitly because from one side one may not clearly see the other. That will make lazy users tend to answer just simply the surface on their side of the road, hoping that it will probably the same on the other, or that it is not that important.

Reply:
It seems to me that it requires two separate quests. Or maybe a single quest with two separate forms would be better - otherwise quest list in preferences would have an apparent duplicate.

Reply:
I'd say two different quests

Reply:
I dropped `cycleway=track`